acceptance/acceptance_test.go

```diff
@@ -439,9 +440,15 @@ func runTest(t *testing.T,
 	// Must be added PrepareReplacementsUser, otherwise conflicts with [USERNAME]
 	testdiff.PrepareReplacementsUUID(t, &repls)
 
-	// User replacements come last:
+	// User replacements:
 	repls.Repls = append(repls.Repls, config.Repls...)
 
+	// Apply these after user replacements in case user replacement capture something like "Job \d+"
```

Comment:
Should we keep this behaviour? I'm afraid otherwise we end up unnecessarily complicating how replacements are applied.

Reply:
+1

With a large and random offset, these replacements are as narrow/focused as they get.

Reply:
It's narrow & focused within local test. However, if you want the same test to run both on local and on cloud than job ids would be different and you need a replacement that is stable regardless.

Reply:
Not a blocker since in that case, you could have two replacements, one for a generic for clouds`\d+` and one for `[TEST_JOB_ID+%d]`. But yeah, I see your point.

Reply:
Ack, thanks.